tests/integration/test_fim/test_registry/test_registry_report_changes/test_registry_diff_size_limit_values.py

```diff
@@ -70,50 +73,44 @@
 
 # Variables
 
-key = "HKEY_LOCAL_MACHINE"
-sub_key_1 = "SOFTWARE\\test_key"
-sub_key_2 = "SOFTWARE\\Classes\\test_key"
+key = WINDOWS_HKEY_LOCAL_MACHINE
+sub_key_1 = MONITORED_KEY
+sub_key_2 = MONITORED_KEY_2
 
 test_regs = [os.path.join(key, sub_key_1),
              os.path.join(key, sub_key_2)]
-
+reg1, reg2 = test_regs
 test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
 wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
-reg1, reg2 = test_regs
-size_limit_configured = 10 * 1024
+size_limit_configured = SIZE_LIMIT_CONFIGURED_VALUE
 
 # Configurations
 
 p, m = generate_params(modes=['scheduled'], extra_params={'WINDOWS_REGISTRY_1': reg1,
```

Comment:
changes the variables in order to representative
```suggestion
conf_params, conf_metadata = generate_params(modes=['scheduled'], extra_params={'WINDOWS_REGISTRY_1': reg1,
```

Reply:
Fixed in commit https://github.com/wazuh/wazuh-qa/pull/2478/commits/50987ed612082b485cdda2af0c6f3c1aeceffed0